pkg/cmd/registry/artifact/artifacts.go

```diff
@@ -18,6 +18,8 @@ func NewArtifactsCommand(f *factory.Factory) *cobra.Command {
 		Use:   "artifact",
 		Short: "Manage Service Registry Artifacts",
 		Long: `
+Manage Service Registry Artifacts using currently selected Service Registry.
```

Comment:
```suggestion
Manage Service Registry Artifacts using the currently selected Service Registry.
```
If it's still technically accurate, consider replacing the "selected Service Registry" with "selected registry".

Reply:
There is huge chance of having another Registry in CLI so trying to be more specific. If we find any docs where we take shortcut it is best to change it